regulations/html.py

```diff
@@ -208,15 +217,31 @@ def makeTOC(articles):
                      regOrGuideLiMatch,
                      regOrGuideLiReplace
                      )
-    ## Numbering/links in the Guidelines for ones that don't correspond to a Regulation.
+
+    ## Replace any remaining 'SEPARATE' label in the Guidelines
+    # FIXME remove this when Guidelines are fixed and updated
```

Comment:
Oh, I just wanted to make sure there was an issue somewhere! Sounds like there are plenty of issues now =)